### ISSUE TITLE
Bug: An image is not working properly and showing the ALT text in the 'Forms and Data Handling' Module in Node JS

### ISSUE THREAD

**Issue by Hriday2406** [TheOdinProject/theodinproject#4888]
### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] The title of this issue follows the `Bug: brief description of bug` format, e.g. `Bug: Lesson complete button does not update on click`
- [X] Would you like to work on this issue?

### Bug description

I was reading the [Forms and Data Handling](https://www.theodinproject.com/lessons/nodejs-forms-and-data-handling#assignment) module in Node JS Course of Java script path.
Just above the assignment section, there should be an image, but it is showing the ALT text and when i click on it, it takes me to a [broken MDN website](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/forms/web_server_form_handling.png). 
![Screenshot](https://github.com/user-attachments/assets/dc16b555-7d77-4a5a-978f-f90bccdb5dda)
![image](https://github.com/user-attachments/assets/f9bc0394-f5fd-4113-b345-3feda9347798)

### How to reproduce

1. Open [Forms and Data Handling](https://www.theodinproject.com/lessons/nodejs-forms-and-data-handling#assignment) module in Node JS Course of Java script path.
2. Navigate to the 'Assignment' Section
3. The broken image is present right above it.

### Expected behavior

1. Open [Forms and Data Handling](https://www.theodinproject.com/lessons/nodejs-forms-and-data-handling#assignment) module in Node JS Course of Java script path.
2. Navigate to the 'Assignment' Section
3. Image explaining the concept should be present right above it.

### What browsers are you seeing the problem on?

Chrome

### What OS are you using?

Windows

### Discord Name

hriday2406

### Additional Comments

I am really enjoying the course and it is teaching me a lot. ♥
I love that there is minimal hand-holding, which is how I like and how it should be.
I was studying and saw this broken image and just thought I should report it.

**Comment by CouchofTomato** [comment]
This was just fixed in another PR I think. It seems to be working now anyway. Let us know if you still have any issues.

**Comment by Hriday2406** [comment]
> This was just fixed in another PR I think. It seems to be working now anyway. Let us know if you still have any issues.

It is fixed now, thanks for the help.
